admin_guide/index.adoc

```diff
@@ -5,5 +5,48 @@
 :icons:
 :experimental:
 
-{product-title} Cluster Administration topics cover the day-to-day tasks for managing
+These Cluster Administration topics cover the day-to-day tasks for managing
 your {product-title} cluster and other advanced configuration topics.
+
+ifdef::openshift-dedicated[]
+WHAT IS THE OPENSHIFT DEDICATED ADMINISTRATOR ROLE?::
+As an administrator of an {product-title} cluster, your account has increased
+permissions and access to all user-created projects. If you are new to the role,
+check out the Getting Started topic on
+link:../getting_started/dedicated_administrators.html[Administering an
+{product-title} Cluster] for a quick overview.
+
+[NOTE]
+====
+_Some configuration changes or procedures discussed in this guide may only be
+performed by the OpenShift Operations Team. They are included in this guide for
+informational purposes to help you as an {product-title} cluster administrator
+better understand what configuration options are possible. If you would like to
+request a change to your cluster that you cannot perform using the administrator
+CLI, please open a support case on the https://access.redhat.com/support/[Red
+Hat Customer Portal]._
+====
+
+When your account has the *dedicated-cluster-admin* authorization role
+link:../architecture/additional_concepts/authorization.html[bound] to it, you
+are automatically bound to the *dedicated-project-admin* for any new projects
+that are created by users in the cluster.
+
+You can perform actions associated with a set of
+link:../architecture/additional_concepts/authorization.html#evaluating-authorization[verbs]
+(e.g., `create`) to operate on a set of
+link:../architecture/additional_concepts/authorization.html#evaluating-authorization[resource
+names] (e.g., `templates`). To view the details of these roles and their sets of
+verbs and resources, run the following:
+
+----
+$ oc describe clusterrole/dedicated-cluster-admin
+$ oc describe clusterrole/dedicated-project-admin
+----
+
+The verb names do not necessarily all map directly to `oc` commands, but rather
+equate more generally to the types of CLI operations you can perform. For
+example, having the `list` verb means that you can see all objects of a given
+resource name, while `get` means that you can see a specific object if you know
+its name.
```

Comment:
The two "see"s are different, right?  My understanding is that `list` displays object names, while `get` displays a named object's details (or "fields" or "keys and values" or ...).